Unit2/Unit2-Project/README.md

```diff
@@ -0,0 +1,22 @@
+# Unit 2 Project: Add an SQL to your full-stack app
+
+##### Credit to Shift_Up coach Adrian Laurenzi
+![Become](https://avatars2.githubusercontent.com/u/38302861?s=200&v=4)
+
+
+### OVERVIEW & GOALS
+
+This project is a follow-on to the Unit1 Project. In this project you will be integrating an SQL database into the full-stack app you built for the Unit 1 project. You will get practice with database design and making a variety of SQL queries in an application context.
+
+
+### SPECIFICATIONS
+
+* Your app must use an SQL-based database system such as MySQL or PostgreSQL. The SQL database will replace Mongo, but we recommend doing this incrementally, so don't worry about fully removing Mongo until you get the SQL database and each of the queries working within your app.
+* Replace each Mongo query with a query to your SQL-based database. You can either use straight SQL or a library such as [Sequelize](https://sequelize.org/).
+* You must have at least 3 tables in your database
+* Ensure each table has a primary key
+* At least one table should have a foreign key constraint to ensure data integrity
+* Add an index to at least one table to improve performance
+* Add functionality to your app to make an SQL query that involves a `JOIN` operation
```

Comment:
Add functionality to your app to make **a** SQL query that involves a `JOIN` operation